<launch>
  <arg name="world_frame" default="world_frame"/>
  <arg name="hotend_group" default="manipulator_tcp"/>
  <arg name="hotend_tcp" default="tcp_frame"/>
  <arg name="robot_model_plugin"/>

  <node name="framefab_process_planning" pkg="framefab_process_planning" type="framefab_process_planning_node"
  output="screen" respawn="true">
    <param name="world_frame" value="$(arg world_frame)"/>
    <param name="hotend_group" value="$(arg hotend_group)"/>
    <param name="hotend_tcp" value="$(arg hotend_tcp)"/>
    <param name="robot_model_plugin" value="$(arg robot_model_plugin)"/>
  </node>
</launch>
